tests/datadriven/test_variants.py

```diff
@@ -97,7 +97,6 @@ def _verifyVariantCallEqual(self, gaCall, pyvcfCall):
         genotype, phaseset = variants.convertVCFGenotype(
             pyvcfCall.data.GT, pyvcfCall.phased)
         # callSetId information is not available in pyvcf.model._Call
-        self.assertIn(pyvcfCall.sample, gaCall.callSetId)
```

Comment:
There's no longer a simple relationship between the sample name and the callSetId. There's no reason there should be, so I removed the check.